packages/kbn-apm-synthtrace/src/lib/apm/processors/get_breakdown_metrics.ts

```diff
@@ -110,6 +110,7 @@ export function getBreakdownMetrics(events: ApmFields[]) {
         'transaction.type': transaction['transaction.type'],
         'transaction.name': transaction['transaction.name'],
         ...pickBy(event, metricsetPicker),
+        ...pickBy(event, instancePicker),
```

Comment:
I think this is duplicated on line 116. we should only need one (and instance doesn't need to be spread twice in the metricset).

Reply:
Changed it https://github.com/elastic/kibana/pull/140256/commits/c8a1788f692ec616e122e667923b80b247ff8421